src/io_utils.py

```diff
@@ -0,0 +1,35 @@
+"""io_utils.py
+
+This module will contain functions pertaining to loading and writing files.
+"""
+import pathlib
+
+import yaml
+
+
+def load_config(fpath: str | pathlib.Path ) -> dict:
```

Comment:
Consider indicating within the function name what kind of config is being loaded (same goes for the docstring below).